ExperimentDatabase.py

```diff
@@ -51,17 +47,30 @@ def setup_experiment(self, name, species, uses_rfid, num_animals, num_groups, ca
                             (name, species, uses_rfid, num_animals, num_groups, cage_max))
         self._conn.commit()
 
-    def setup_groups(self, group_names, animals_per_group):
+    def setup_groups(self, group_names):
         '''Adds the groups to the database.
             arg1 (list): a list of all the group names
-            arg2 (int): a number representing the number of animals per group
         '''
         for group in group_names:
-            self._c.execute(''' INSERT INTO groups (name, num_animals) 
-                                VALUES (?, ?)''',
-                                (group, animals_per_group))
+            self._c.execute(''' INSERT INTO groups (name, num_animals, full) 
+                                VALUES (?, ?, ?)''',
+                                (group, 0, 0))
             self._conn.commit()
 
+    def setup_cages(self, num_animals, num_groups, cage_max):
+        cages_per_group = (int(num_animals)/int(num_groups)) / int(cage_max)
```

Comment:
Shall we use integer division instead of float? Or do we need float division mandatorily?

Reply:
I will switch it to integer divison